packages/cashier/src/types/stores/index.ts

```diff
@@ -1,3 +0,0 @@
-import { useStore } from '@deriv/stores';
-
-export type TRootStore = ReturnType<typeof useStore>;
```

Comment:
@hamid-deriv I think we still need it 🤔 Any reason you have deleted this file?

Reply:
I have added this to the new file but I didn't push it. thanks for commenting this :pray: